source/agora/consensus/EnrollmentManager.d

```diff
@@ -587,4 +672,13 @@ unittest
     man.getUnregisteredEnrollments(enrolls);
     assert(enrolls.length == 3);
     assert(enrolls.isStrictlyMonotonic!("a.utxo_key < b.utxo_key"));
+
+    // test serialization/deserializetion for pre-images
+    auto preimages_1 = man.getPreimages();
```

Comment:
You can access the field directly, because private allows access from the same module.

So just `auto preimages_1 = man.preimages;`.